src/components/layout/WithVehicleReference.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { Center, chakra, Grid, GridItem } from '@chakra-ui/react';
+
+import VehicleReference, { VehicleProps } from '../vehicleReference';
+import Stack from '../stack';
+import Link from '../link';
+import { ArrowLeftIcon } from '../icons';
+
+interface Props {
+  title: string;
+  backLink?: {
+    text: string;
+    url: string;
+  };
+  vehicle: VehicleProps;
+}
+
+const LayoutWithVehicleReference: FC<PropsWithChildren<Props>> = ({
+  title,
+  backLink,
+  vehicle,
+  children,
+}) => {
+  const Component = chakra('main');
+
+  return (
+    <Center>
+      <Component maxWidth="container.xl">
+        <Grid
+          templateAreas={{
```

Comment:
This does not match the Figma file, the header is aligned with the image and the backlink takes the whole width
![Screenshot 2022-08-03 at 12 19 03](https://user-images.githubusercontent.com/3371760/182585454-1a725dac-5a51-40b7-9296-67c4376f7304.png)

Reply:
Not fully sure but I think you need to specify a 12 gird to match the design, something along this lines maybe https://codepen.io/legatore/pen/rNddBNM (estimated the gird gap)
(gen: https://grid.layoutit.com/?id=YvDcDKe)
![image](https://user-images.githubusercontent.com/3371760/182588955-02aa8db4-8827-4ed6-8542-4243cb605b68.png)

Reply:
done